docs/source/concepts/PipInstall.rst

```diff
@@ -0,0 +1,72 @@
+.. _PipInstall:
+
+Pip Install
+===========
+
+What is Pip?
+------------
+
+``pip`` is a package manager for Python that is primarily used for installing Python packages
+from the Python Package Index (PyPI) and other package repositories. It is the default package
+manager for Python and is commonly used for managing Python libraries and dependencies.
+
+What is Conda?
+--------------
+
+``Conda`` is a package management tool that is not specific to Python. It is a cross-platform
+package manager and environment management system that can handle packages written in different
+programming languages, not just Python. Conda is the dependency system used in Mantid.
+
+Can they be used together?
+--------------------------
+
+While it is possible to use ``pip`` and ``Conda`` together, there are some potential issues and
+complexities when doing so:
+
+1. **Dependency Conflicts:** When you use both ``pip`` and ``Conda`` in the same environment,
+   there can be conflicts between the installed packages. These conflicts can lead to unexpected
+   behavior and errors.
+
+2. **Package Versions:** ``pip`` and ``Conda`` might have different versions of the same package,
+   and managing which version takes precedence can be challenging.
+
+3. **Environment Isolation:** Conda is designed to create isolated environments, which is
+   useful for managing dependencies for different projects. Mixing ``pip`` and ``Conda`` in the
+   same environment can undermine this isolation.
+
+For these reasons, Mantid does not install any ``pip`` dependencies automatically. If you require
+a PyPi package for your workflow, you can install the package manually. Note that you do so at
+your own risk.
+
+.. _pip-install-ref:
+
+How to extend Mantid with a Pip install?
```

Comment:
I think it is better to make it clear which method to use in every platform:
First method: if you are using mantid from a conda build
Second method: if you are using GUI in other platforms

Reply:
Thanks, I've added recommended OS's for each of the options

Users shouldn't need to use `pip install` for a non-conda build because the standalone installers do the pip installations for them

Reply:
oh, I thought it is a general issue